docs/product-development-guide/src/main/asciidoc/assembly_security-management.adoc

```diff
@@ -0,0 +1,66 @@
+[id='assembly_security-management']
+= Security Management
+
+{PRODUCT} {RELEASE} introduces enhanced security features for Business Central that enable administrators to specify the home perspective for users, as well as control access to assets. For a detailed description of these new features, see the "Business Central Settings" section of the _{PRODUCT} User Guide_. 
+
+The set of permissions assigned to every user, role, and group is called the authorization or security policy. Every application contains a security policy which is used each time the system checks a permission. The security policy is stored in the `.niogit` file in the `security.git` repository.
```

Comment:
Add a comma before "which is used each time..."

Reply:
'Which' in this case is used to introduce a restrictive clause. To remove the clause would change the meaning of the sentence, therefore a comma is not used (only for restrictive clauses).